tests/test_embedded_files.py

```diff
@@ -0,0 +1,227 @@
+"""Test the EmbeddedFile setters and read-only name property."""
```

Comment:
The docstring seems odd. Additionally, why aren't you re-using the existing test file at https://github.com/py-pdf/pypdf/blob/main/tests/generic/test_files.py?